### ISSUE TITLE
Does DH provide a way of storing attributes of a row/object but never displaying them?

### ISSUE THREAD

**Issue by turbomam** [cidgoh/DataHarmonizer#373]
@pkalita-lbl an example case would be associating  NMDC minted `id`s with entries in the Submission Portal, but not showing them

**Comment by pkalita-lbl** [comment]
No, or at least not in a particularly clean way. That use case was called out in this proposal: https://github.com/cidgoh/DataHarmonizer/pull/359 but that hasn't really gotten much traction yet.

**Comment by ddooley** [comment]
DH should have this, but a question is whether this could be coded in the LinkML spec directly?  So generically, not specific to DH user interface?  This gets back to question of how we should provide a mechanism for generically (i.e. DH or any other app) hiding classes from a user interface menu choices and slots from DH interface columns (from a given schema).

What if we had a class and slot slot called "ui_behaviour" and we start with a basic "hide" value.

**Comment by ddooley** [comment]
Or maybe "ui_behaviour" is placed in a class or slot annotation that DH pays attention to?

**Comment by ddooley** [comment]
I think this basically entails "hidden" column capability, which taking "read-only" column concept one step further via a LinkML specification about what to show or not in a given user interface.  So I'll close this and continue discussion in https://github.com/cidgoh/DataHarmonizer/issues/337
